src/app/(content-pages)/page.tsx

```diff
@@ -139,7 +139,22 @@ const LandingPage: FC = () => {
             text: 'Community groups and organizations are taking action.',
           }}
           body={{
-            text: 'Community groups have been cleaning up lots in their own neighborhoods for decades. Large organizations like the Pennsylvania Horticulture Society have cleaned, greened and now maintain thousands of lots. Their efforts have been instrumental in proving this approach works.',
+            text: (
+              <>
+                Community groups have been cleaning up lots in their own
+                neighborhoods for decades. Large organizations like the
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer nofollow"
+                  href="https://phsonline.org/"
+                  className="default"
+                >
+                  Pennsylvania Horticultural Society
```

Comment:
Add space between "the" and "Pennsylvania."

![Screen Shot 2024-09-23 at 11 44 00 PM](https://github.com/user-attachments/assets/35f88db7-0806-41bd-9a53-cacc85d8526a)